lib/utils/messages.ts

```diff
@@ -9,19 +9,16 @@ create a new one at: https://github.com/balena-io/balena-cli/issues/\
 `;
 
 const debugHint = `\
-Additional information may be available by setting a DEBUG=1 environment
-variable: "set DEBUG=1" on a Windows command prompt, "$env:DEBUG = 1" on
-powershell, or "export DEBUG=1" on Linux or macOS.\n
+Additional information may be available with the \`--debug\` flag.
```

Comment:
A few lines above, there is a `reachingOut` message that apparently is still printed by the `balena login` command. Removing or shortening that message seems to fit under the objective of issue #1779, and this PR/commit is labeled with "Resolves #1779".

Reply:
Or not.

Reply:
Yeah I left that one in on purpose, because it seemed like some friendly output might be nice  and didn't seem to hurt there.  Perhaps let's leave it for this PR and see what people thing (will be a quick change to take it out later).